Issue building with webpack

After adding this dependency i'm seeing an error:

```
ERROR in ./~/cb2promise/index.js
Module not found: Error: Cannot resolve 'file' or 'directory' /Users/johnryan/Desktop/app/node_modules/cb2promise/lib/cb2promise in /Users/johnryan/Desktop/app/node_modules/cb2promise
 @ ./~/cb2promise/index.js 3:17-55
```

and

```
ERROR in ./~/coffee-script/bin/coffee
Module parse failed: /Users/johnryan/Desktop/app/node_modules/coffee-script/bin/coffee Unexpected character '#' (1:0)
You may need an appropriate loader to handle this file type.
SyntaxError: Unexpected character '#' (1:0)
```

Is there a recommended solution for this?

--- a/package.json
+++ b/package.json
@@ -3,7 +3,7 @@
   "description": "Converts whatever standard NodeJS callback function into ES6 standard promise.",
   "homepage": "https://github.com/Kikobeats/cb2promise",
   "version": "1.0.3",
-  "main": "./index.js",
+  "main": "./dist/cb2promise.js",
   "author": {
     "name": "Kiko Beats",
     "email": "josefrancisco.verdu@gmail.com",

--- a/test/test.coffee
+++ b/test/test.coffee
@@ -1,4 +1,4 @@
-cb2promise = require '..'
+cb2promise = require '../index.js'
 should     = require 'should'
 sampleFunction = (done) -> done null, 'hello world'
 

